### PR TITLE
feat: Use embed timestamp in modpings off

### LINKED ISSUE #1547: Use embed timestamp in mod pings off
When a mods turns off mod pings, a confirmation is sent to inform the user that their pings have successfully been turned off.

In this confirmation, we currently include the time at which it is due to be sent, this time is in UTC.

I propose we refactor this part of the code to instead use an Embed, with a the timestamp field.
https://github.com/python-discord/bot/blob/ce819ade482e82ecbc474bce5fb8ac9dd8b37b40/bot/exts/moderation/modpings.py#L107
This would mean that the time would automatically get converted to the user's current time zone by Discord.

### PR DESCRIPTION
**Pull request by ToxicKidz**
## Relevant Issues
Closes #1547 

## Description
This PR makes it so that when the bot sends the mod pings off embed, it uses and embed timestamp to show when the role will be given back, so that it can be converted to local readable time, instead of using an ISO format.

### CODE REVIEW

File: bot/exts/moderation/modpings.py [hunk 1]
```diff
@@ -104,7 +104,9 @@ async def off_command(self, ctx: Context, duration: Expiry) -> None:
             self._role_scheduler.cancel(mod.id)
         self._role_scheduler.schedule_at(duration, mod.id, self.reapply_role(mod))
 
-        await ctx.send(f"{Emojis.check_mark} Moderators role has been removed until {until_date}.")
+        embed = Embed(timestamp=duration, colour=Colours.bright_green)
+        embed.set_footer(text="Moderators role has been removed", icon_url=Icons.green_checkmark)
```
Comment (ChrisLovering) [inline bot/exts/moderation/modpings.py:108]:
```suggestion
        embed.set_footer(text="Moderators role has been removed until", icon_url=Icons.green_checkmark)
```
I'm not sure what other's think, but I preferred the wording you have before, with `until` here. It adds context to the date.

Comment (mbaruh) [reply bot/exts/moderation/modpings.py:108]: We could also move the current footer to the embed's text, and just have "Until" in the footer.

Comment (ToxicKidz) [reply bot/exts/moderation/modpings.py:108]: > I'm not sure what other's think, but I preferred the wording you have before, with `until` here. It adds context to the date.

I removed it by accident, my bad.

Comment (mbaruh) [reply bot/exts/moderation/modpings.py:108]: Just adding back until seems ok, on second thought

Comment (ToxicKidz) [reply bot/exts/moderation/modpings.py]: Resolved in https://github.com/python-discord/bot/pull/1556/commits/5c1cf2b5fd94359d42d147ce8687dd27b05e193d.

**Review by ChrisLovering — APPROVED** [review]

**Review by Akarys42 — APPROVED** [review]

**Pull request merged**